modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/container/request/filter/ContextContainerRequestFilter.java

```diff
@@ -100,6 +105,15 @@ public void filter(ContainerRequestContext containerRequestContext) {
 			containerRequestContext, PhaseInterceptorChain.getCurrentMessage());
 	}
 
+	@Override
```

Comment:
I think we should update the commit message:

"The issue only occurs when the request fails because of an invalid siteId." <<-- I think this is not totally true

Reply:
Yes, I will do so.

Reply:
See 710d30ea471740065a2fcdbd0c9e476ef5123ffd.